bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -127,6 +135,12 @@ pub mod pallet {
 			message: Xcm<()>,
 			message_id: XcmHash,
 		},
+		RewardTipAdded {
+			origin: Location,
+			message_id: MessageId,
+			asset: Asset,
+			xcm_message_id: XcmHash,
+		},
```

Comment:
See comment below,  we should just emit `MessageSent`, like in `register_token`. All this event mostly does is repeat the input arguments to the extrinsics.

```suggestion
```